internal/usecase/util/access_token/access_token_test.go

```diff
@@ -0,0 +1,51 @@
+package access_token_test
+
+import (
+	"testing"
+	"time"
+	"todoapp/internal/entity"
+	"todoapp/internal/usecase/util/access_token"
+)
+
+func TestCreateToken(t *testing.T) {
+	user := entity.User{
+		Username: "daopham",
+	}
+
+	_, err := access_token.CreateToken(&user, time.Now())
+
+	if err != nil {
+		t.Errorf("expect nil err, got %v", err)
+	}
+}
+
+func TestParseToken(t *testing.T) {
+	user := entity.User{
+		Username: "daopham",
+	}
+	tokenStr, _ := access_token.CreateToken(&user, time.Now())
+
+	_, err := access_token.ParseToken(tokenStr)
+
+	if err != nil {
+		t.Errorf("expect nil err, got %v", err)
+	}
+}
+
+func TestExtractToken(t *testing.T) {
```

Comment:
combine this with TestParseToken to TestVerify

TestVerify for success and failure cases

Reply:
success: valid time

failure: time expired, wrong hash